src/Compiler/Microsoft.AspNetCore.Razor.Language/src/RazorCodeGenerationOptionsBuilder.cs

```diff
@@ -95,6 +95,11 @@ public abstract class RazorCodeGenerationOptionsBuilder
     /// </summary>
     internal bool SuppressAddComponentParameter { get; set; }
 
+    /// <summary>
+    /// Determines if the file paths emitted as part of line pragmas should be mapped back to a valid path on windows.
+    /// </summary>
+    internal bool RemapDesignTimeLinePragmaPathsOnWindows { get; set; }
```

Comment:
It looks like there's no check for design-time, this option is applied always. I think it should not have "design time" in its name.

```suggestion
    internal bool RemapLinePragmaPathsOnWindows { get; set; }
```